Mail2BugUnitTests/Mocks/Email/IncomingEmailMessageMock.cs

```diff
@@ -65,6 +65,7 @@ public static IncomingEmailMessageMock CreateWithRandomData(int numAttachments)
         public string Subject { get; set; }
         public string RawBody { get; set; }
         public string PlainTextBody { get; set; }
+        public string ConversationGuid { get { return this.ConversationIndex.Substring(12, 32); } }
```

Comment:
Does this accurately mock ConversationId?  From the information you've shown me, @vitru, it's always 16 bytes long, starting at byte 6.